pkgs/by-name/sc/scram_2/package.nix

```diff
@@ -0,0 +1,85 @@
+{
+  lib,
+  stdenvNoCC,
+  fetchFromGitHub,
+  makeWrapper,
+  perl,
+}:
+let
+  escapeMakeFlag = builtins.replaceStrings [ "\$" ] [ "\$\$" ];
+in
+stdenvNoCC.mkDerivation (finalAttrs: {
+  pname = "scram";
+  version = "2.2.9-pre14";
+
+  __structuredAttrs = true;
+
+  SCRAM_VERSION = "V${
+    builtins.replaceStrings
+      [
+        "."
+        "-"
+      ]
+      [
+        "_"
+        "_"
+      ]
+      finalAttrs.version
+  }";
```

Comment:
```suggestion
  SCRAM_VERSION = "V${lib.replaceStrings [ "." "-" ] [ "_" "_" ] finalAttrs.version}";
```

Reply:
`nixfmt-rfc-style` insists on expanding it across multiple lines. How can we force `nixfmt-rfc-style` to leave it unchanged?

Reply:
Update: Rebasing on top of the `master` branch seems to fix the formatter issue.